2020-01-Winter/4_trees/README.md

```diff
@@ -23,21 +23,56 @@ Problems and solutions for Trees session on February 14, 2020.
 
 <a name="p1"/>
 
-### 1. PROBLEM 1 TODO :bug:
+### 1. Path Sum
 
-Source: TODO :bug:
+Source: [Leetcode] https://leetcode.com/problems/path-sum/
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Problem Statement: Given a binary tree and a sum, determine if the tree has a root-to-leaf path such that adding up all the values along the path equals the given sum.
 
-#### Example Input
+Note: A leaf is a node with no children.
 
-If the problem is simple enough, remove this section. TODO :bug:
+#### Example Input
+Example 1:
+
+Given the below binary tree and sum = 22,
+
+      5
+     / \
+    4   8
+   /   / \
+  11  13  4
+ /  \      \
+7    2      1
+return true, as there exist a root-to-leaf path 5->4->11->2 which sum is 22.
```

Comment:
For any sections where monospaced formatting matters, I _strongly_ recommend enclosing the lines in backticks.

Right now, the file looks like:
![funky_formatting](https://user-images.githubusercontent.com/29528527/74423123-97182880-4e04-11ea-9399-8071254a088f.png)

By enclosing the tree segment in 3 backticks, you'll get the structure of the text preserved, including leading spaces. I will add them as suggestions you can commit.